### DISCUSSION TITLE
part-ii/2018-19/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#258] [Question comments]
Part II 2018 Asymptotic Methods: Paper 2, Section II, B 

https://questions.tripos.org/part-ii/2018-19/

**Comment by acornnroca** [comment]
<details><summary>first bit</summary>

take contours in wedges between real axis and -pi/4 below ( -pi/6 for second integral)

</details>

<details><summary>$$Q_\alpha$$</summary>

$$Q_alpha = \cos(x- \alpha \pi /2 - \pi /4) \sqrt{2/x \pi}$$

</details>

<details><summary>$$Q_x$$</summary>

$$Q_x = 1 / \pi 6^{1/3} x^{-1/3} \Gamma ( 4/3) \cos{\pi / 6}$$.

</details>
